src/packages/solid/components/ListItem/ListItem.styles.ts

```diff
@@ -0,0 +1,178 @@
+  Title: TextStyleSet;
+  Description: TextStyleSet;
+}
+
+export type ListItemStyleProperties = {
+  alpha?: NodeStyles['alpha'],
+  paddingX?: number,
+  paddingY?: number,
+  contentSpacing?: number,
+  titleTextStyle?: object,
+  descriptionTextStyle?: object
+}
+
+type ListItemConfig = ComponentStyleConfig<ListItemStyleProperties>;
+
+/* @ts-expect-error next-line themes are supplied by client applications so this setup is necessary */
+const { ListItem: { tone: defaultTone, ...themeStyles } = { style: {} } } = theme?.componentConfig;
+/* @ts-expect-error next-line themes are supplied by client applications so this setup is necessary */
+const { Button: { tone: buttonDefaultTone, ...buttonThemeStyles } = { style: {} } } = theme?.componentConfig;
+/* @ts-expect-error next-line see above */
+const { Surface: { tone: surfaceDefaultTone, ...surfaceThemeStyles } = { style: {} } } = theme?.componentConfig;
+
+const container: ListItemConfig = {
+  themeKeys: {
+    alpha: 'alpha',
+    paddingX: 'paddingX',
+    paddingY: 'paddingY',
+    contentSpacing: 'contentSpacing',
+    titleTextStyle: 'titleTextStyle',
+    descriptionTextStyle: 'descriptionTextStyle'
+  },
+  base: {
+    height: theme.typography.headline3.lineHeight + theme.typography.body3.lineHeight + theme.spacer.xl * 2,
+    paddingY: theme.spacer.xl, 
+    paddingX: theme.spacer.xl, 
+    color: theme.color.interactiveNeutral,
+    borderRadius: theme.radius.sm,
+    contentColor: theme.color.fillNeutral,
+    width: getWidthByUpCount(theme, 3)
+  },
+  modes: {
+    focus: {
+      color: theme.color.interactiveNeutralFocus,
+      contentColor: theme.color.fillInverse
+    },
+    disabled: {
+      color: theme.color.fillNeutralDisabled
+    }
+  },
+  tones: {
+    inverse: {
+      color: theme.color.interactiveInverse
+    },
+    brand: {
+      color: theme.color.interactiveBrand,
+      focus: {
+        color: theme.color.fillNeutral
+      }
+    }
+  },
+  themeStyles: {
+    ...surfaceThemeStyles, 
+    ...buttonThemeStyles,
+    ...themeStyles
+  }
+}
+
+const title: ListItemConfig = {
+  themeKeys: {},
+  base: {
+    color: theme.color.textNeutral,
+    textAlign: 'left',
+    x: container.base.paddingX,
+    y: container.base.paddingY,
+    ...theme.typography.headline3
+  },
+  modes: {
+    focus: {
+      color: theme.color.textInverse
+    },
+    disabled: {
+      color: theme.color.textNeutralDisabled
+    }
+  },
+  tones: {
+    inverse: {
+      color: theme.color.fillNeutral
+    },
+    brand: {
+      color: theme.color.fillBrand,
+      focus: {
+        color: theme.color.fillBrand
+      }
+    }
+  },
+  themeStyles: {
+    ...surfaceThemeStyles, 
+    ...buttonThemeStyles,
+    ...themeStyles
+  }
+}
+
+const description: ListItemConfig = {
+  themeKeys: {
+    color: 'textColor'
+  },
+  base: {
+    color: theme.color.textNeutral,
+    textAlign: 'left',
+    x: container.base.paddingX,
+    y: title.base.lineHeight + title.base.y,
```

Comment:
i think we'll need to find another way to accomplish this - as this is written it doesn't support tones or componentConfig styles. are these themeable in LUI2? if not i think we should calculate these x and y values inside the component code

Reply:
The `x` value is themeable in LUI, but the Y is not. All of the content in ListItem is just centered vertically.